fv3core/.jenkins/actions/run_regression_tests.sh

```diff
@@ -12,8 +12,16 @@ make get_test_data
 
 if [ ${python_env} == "virtualenv" ]; then
     export TEST_ARGS="${TEST_ARGS} --junitxml=${JENKINS_DIR}/${XML_REPORT}"
-    CONTAINER_CMD="srun" make tests savepoint_tests
+    if [[ ${FV3_DACEMODE} == "True" ]]; then
+        CONTAINER_CMD="srun" make savepoint_tests
```

Comment:
If you make more edits to this file, can you move some logic out of the virtualenv if-statement by having it control exporting environment variables only, and then having one line (or two, if you need the dacemode if-statement) calling `make`? Two `make` lines is maybe OK but four is getting up there.

Reply:
I'm keeping this for now because passing `CONTAINER_CMD` and `VOLUMES` to makefile is quite difficult unless it's explicitly passed like `CONTAINER_CMD="srun" make savepoint_tests`. But then you need to have more if statements to be able to run for both daint and gce. Since there's a chance that we don't even run `make tests` (comments above), maybe it's not an issue here.

Reply:
I thought Makefiles read environment variables? e.g. if you `export CONTAINER_CMD="srun"` then `make savepoint_tests`, it's the same as if you `CONTAINER_CMD="srun" make savepoint_tests`, apart from that after the command is run the environment variable will still be set (which shouldn't break anything here).

Reply:
You're right, I must have missed `export` because it didn't work when I tried yesterday. Updating this.